arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,578 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagementClient",
+    "version": "2015-10-31"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
```

Comment:
in 'consumes' it's not really valuable to add `text/json` -- `application/json` is sufficient.